src/client/mod.rs

```diff
@@ -119,6 +120,8 @@ mod twin;
 
 static mut IOTHUB_INIT_RESULT: i32 = -1;
 static IOTHUB_INIT_ONCE: Once = Once::new();
+static DO_WORK_FREQUENCY_IN_MS: &str = "DO_WORK_FREQUENCY_IN_MS";
```

Comment:
i would prefer a azure sdk context. e.g. when setting it in a service file, it's not clear that this only refers to the azure sdk worker loop-
e.g. `AZURE_SDK_DO_WORK_FREQUENCY_IN_MS`

Reply:
done